src/compile.ts

```diff
@@ -573,6 +573,12 @@ export function compile(
           node.declarationList.declarations.length === 1
         ) {
           return toExpr(node.declarationList.declarations[0], scope);
+        } else if (ts.isVariableDeclarationList(node)) {
+          return newExpr("VariableList", [
```

Comment:
Shouldn't it be called `VariableDeclList`?

Reply:
It could, we called `VariableDeclaration` a `VariableStmt`, so I thought `VariableList` was more neutral.